esp/bindings/http/platform/httpservice.cpp

```diff
@@ -1230,6 +1235,33 @@ EspAuthState CEspHttpServer::authNewSession(EspAuthRequest& authReq, const char*
     return authSucceeded;
 }
 
+void CEspHttpServer::sendAuthorizationMsg(EspAuthRequest& authReq)
+{
+    StringBuffer resp;
+    const char* errMsg = authReq.ctx->getRespMsg();
+    ESPSerializationFormat format = m_request->queryContext()->getResponseFormat();
+    if (format == ESPSerializationJSON)
+    {
+        resp.set("{ ");
+        resp.append(" \"LoginResponse\": { ");
+        if (isEmptyString(errMsg))
+            resp.append("\"Error\": \"Access Denied.\"");
+        else
+            resp.appendf("\"Error\": \"%s\"", errMsg);
+        resp.append(" }");
+        resp.append(" }");
```

Comment:
Would it make sense to combine some of these appends ?

Reply:
It shows the JSON layers better to keep those appends.